spec/system/admin_customize_themes_spec.rb

```diff
@@ -6,8 +6,20 @@
   fab!(:admin)
 
   let(:admin_customize_themes_page) { PageObjects::Pages::AdminCustomizeThemes.new }
-
-  before { sign_in(admin) }
+  let(:translations_settings) do
+    theme.set_field(
+      target: :translations,
+      name: "yaml",
+      value: File.read("#{Rails.root}/spec/fixtures/theme_settings/translations_settings.en.yaml"),
```

Comment:
You need to use the language as the name here: 

```suggestion
      name: "en",
      value: { en: { group: { hello: "Hello there!" } } }.deep_stringify_keys.to_yaml
```

And convert a Ruby object to YAML is nice to see in context. 

Also, further below, `theme.translations[:translations_settings]` will give you an error, I think you can just return `theme.translations`.